# Thinking in Cobiont Space

This session is part of [**Biodiversity Genomics Academy 2024**](https://thebgacademy.org/)

[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/thebgacademy/thinking-in-cobiont-space)

[YouTube Recording](https://www.youtube.com/@thebiodiversitygenomicsacademy) <-- To be updated after BGA24

## Session Leader(s)
Amjad Khalaf - PhD Student in DTOL, Sanger


## Description

Typically, when an organism is sequenced, you don't just sequence the organism of interest, you also sequence its lunch, its commensals, and its parasites - i.e. its cobionts. In this session, we will look at methods and best practices to isolate cobiont reads from a metagenomic sample, assemble them, and assess the resulting genome assembly quality. We will also explore how data mining can be carried out for cobionts of interest, and what kinds of biological insights have been possible from such efforts and cobiont genome assemblies. Tools we will use include BlobToolKit, minimap2, mummer, hifiasm, and the interactive dot plotter HyraxDotPlot.

## Prerequisites

1. Understanding of linux command line basics

>[!warning] "Please make sure you MEET THE PREREQUISITES and READ THE DESCRIPTION above"
>
>    You will get the most out of this session if you meet the prerequisites above.
>
>    Please also read the description carefully to see if this session is relevant to you.
>    
>    If you don't meet the prerequisites or change your mind based on the description or are no longer available at the session time, please email damon at thebgacademy.org to cancel your slot so that someone else on the waitlist might attend.


## Instructions
For Session Leader
